src/app/screens/Home/index.tsx

```diff
@@ -18,12 +18,15 @@ import { useNavigate } from "react-router-dom";
 import { toast } from "react-toastify";
 import browser from "webextension-polyfill";
 import api from "~/common/lib/api";
+import utils from "~/common/lib/utils";
 import type { Allowance, Battery, Transaction } from "~/types";
 
 dayjs.extend(relativeTime);
 
 function Home() {
   const [allowance, setAllowance] = useState<Allowance | null>(null);
+  const [blocklist, setBlocklist] = useState<boolean | null>(null);
```

Comment:
Can we call this `isBlocked`? That indicates it's a boolean and that this website is blocked.  
"if blocklist than show button" is a bit unclear.